hw/top_earlgrey/data/ip/chip_sysrst_ctrl_testplan.hjson

```diff
@@ -5,77 +5,191 @@
   name: chip_sysrst_ctrl
   testpoints: [
     {
-      name: chip_sw_sysrst_ctrl_combo_reset
-      desc: '''Verify that sysrst_ctrl issue reset on combo detection
+      name: chip_sw_sysrst_ctrl_reset
```

Comment:
Looking at the change, this testplan entry was originally talking about detecting specific combos. Are we dropping that intentionally?

Could you take a quick look at the existing DV vseq etc. and check what it was actually doing?

Reply:
This is just the name of the testpoint, the test itself is the same and uses combos: https://github.com/lowRISC/opentitan/blob/master/sw/device/tests/sim_dv/sysrst_ctrl_reset_test.c#L40

Reply:
That makes sense. The testplan before this change talks about a specific feature (combo detect) and it no longer talks about the feature after the change.

Is there a way to tweak the "end text" so that it still talks about combo detection, which would justify what the firmware in the test is doing?

Reply:
Aargh. Yet another example of me getting confused about duplicate files! What you've done looks completely sensible.